indico/modules/events/contributions/controllers.py

```diff
@@ -107,7 +123,10 @@ def _process(self):
         if form.validate_on_submit():
             contrib = create_contribution(self.event_new, form.data)
             flash(_("Contribution '{}' created successfully").format(contrib.title), 'success')
-            return jsonify_data(html=_render_contribution_list(self.event_new))
+            flash(_("Note that the contribution '{}' might not be displayed in the list due to the enabled filters")
```

Comment:
this should be shown only if there are filters enabled and the contribution is actually being filtered